WooCommerce/UITestsFoundation/Screens/Orders/SingleOrderScreen.swift

```diff
@@ -8,11 +8,24 @@ public final class SingleOrderScreen: ScreenObject {
 
     init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
-            expectedElementGetters: [ { $0.staticTexts["summary-table-view-cell-title-label"] } ],
+            expectedElementGetters: [{ $0.staticTexts["summary-table-view-cell-title-label"]} ],
             app: app
         )
     }
 
+    @discardableResult
+    public func verifySingleOrder(order: OrderData) throws -> Self {
+        app.assertTextVisibilityCount(textToFind: order.status, expectedCount: 1) //checks that the Mock order status appears on the screen
```

Comment:
Is the comment after this line intentional? Maybe better to put on a separate line before it if needed (same comment for the few lines below as well):

```suggestion
        // checks that the Mock order status appears on the screen
        app.assertTextVisibilityCount(textToFind: order.status, expectedCount: 1) 
```

Reply:
I'd vote for removing the comment, because the assertion makes it pretty clear that we expect to see the order status once.